src/interfaces/ProjectInterfaces/CostCenter/validation-costCenter.ts

```diff
@@ -0,0 +1,45 @@
+import * as yup from 'yup';
+import CostCenterType, { Type } from './costCenterType';
+
+export const CostCentersSchema = yup.object().shape({
+    name: yup
+        .string()
+        .required("Name is required")
+        .max(100, "Name Can't excced 100 Letters"),
+
+    nameSecondLanguage: yup
+        .string()
+        .required("Name is required")
+        .max(100, "Name Can't excced 100 Letters"),
+
+    nodeType: yup
+        .mixed<Type>()
```

Comment:
can it be NodeType (enum) instead of number to be more cleaner
@diaa852000

Reply:
handled

Reply:
done